_blog-src/_posts/2015-07-07-fuse-apis.markdown

```diff
@@ -0,0 +1,171 @@
+</ns0:Customer>
+{% endhighlight %}
+
+Configure apiman to manage Fuse endpoint
+========================================
+
+To install the latest version of apiman follow this [guide](http://www.apiman.io/latest/download.html). The installation process is again very simple, all you need to do is to extract apiman overlay zip on top of [Wildly 8](http://www.wildfly.org/) server. 
+
+After completing the installation, start apiman by executing the follogin command from `<wildfly>/bin` folder:
+
+{% highlight sh %}
+./standalone.sh -c standalone-apiman.xml
+{% endhighlight %}
+
+### Quick public service setup
+
+For demonstration purposes I'll be creating a Public Service, however in real life you might want to configure different plans, various contracts, etc. 
+Please consult the apiman [user guide](http://www.apiman.io/latest/user-guide.html) for more details.
+
+In order to manage APIs in apiman you need to create a new Organization to which your APIs will belong. 
+You can do that easily in apiman manager, typically available at: [http://localhost:8080/apimanui](http://localhost:8080/apimanui).
+
+Once logged in apiman manager, locate the Organizations sections on the initial page, select _Create a New Organization_ link, provide a name and hit _Create Organization_ button. This will take you to organization details page where you can create a new Service by clicking on _New Service_ button under _Services_ tab:
+
+![Image: Create Service](/blog/images/2015-07-07/service.png)
+
+You will be asked to provide a name and a version for this Service. Once the service is successfully created there are few things remaining before you can publish and start using it. 
+First, you need to provide an implementation base url on the _Implementation_ tab. Use the base URL of the REST Quickstart and save your changes:
+
+![Image: Provide Endpoint Implementation](/blog/images/2015-07-07/implementation.png)
+
+Second, you may want to apply some policies to this Publis Service. To do that, go to Policies tab and configure a policy of your choice. 
+I used the Rate Limiting policy to limit usage of this service to 5 per minute:
+
+![Image: Assign policy](/blog/images/2015-07-07/policy.png)
+
+Now you are all set to make the service public by checking _Make this service public_ under _Plans_ tab, saving the changes and clicking the _Publish_ button (which should be now enabled).
+
+![Image: Publish Service](/blog/images/2015-07-07/publish.png)
+
+After you have published the service, have a look at the _Endpoint_ tab to look up the URL to be used to invoke this public service:
+
+![Image: Managed Service Endpoint](/blog/images/2015-07-07/endpoint.png)
+
+Testing your setup
+==================
+
+Depending on which policy you assinged to the Public Service you might experience different behavior. However, if you followed the tutorial and assigned the Rate Limiting policy after reaching the maximum number of allowed request you will get a similar output:
```

Comment:
- allowed request you will get a similar output
- allowed requests you will get an output similar to: